uber/config.py

```diff
@@ -136,6 +136,22 @@ def SUPPORTER_COUNT(self):
                                                 sa.Attendee.amount_paid >= self.SUPPORTER_LEVEL).count()
             return individual_supporters + group_supporters
 
+    @property
+    def SQLALCHEMY_URL(self):
+        """
+        support reading the DB connection info from an environment var (used with Docker containers)
+        example env vars:
+        DB_PORT_5432_TCP_ADDR="172.17.0.8"
+        DB_PORT_5432_TCP_PORT="5432"
+        """
+        docker_db_addr = os.environ.get('DB_PORT_5432_TCP_ADDR')
+        docker_db_port = os.environ.get('DB_PORT_5432_TCP_PORT')
```

Comment:
If we end up doing a lot of this, we could modify ``__getattr__`` to automatically check ``os.environ``, e.g. you could jsut say ``c.DB_PORT_5432_TCP_ADDR`` and have that automatically check the environment.  Not necessary for this changeset, but something we should think about as we start to access environment variables for more and more things.